app/rules/school.js

```diff
@@ -53,7 +53,7 @@ CATS.Rule.School = Classify(CATS.Rule.Base, {
         var teams_arr = _.map(teams, function (v, team_id) {
             var points_cnt = 0;
             for (var i = 0; i < contest.problems.length; ++i)
-                points_cnt += teams_problems[team_id][i].points;
+                points_cnt += parseFloat(teams_problems[team_id][i].points);
```

Comment:
It is called earlier in ioinformatics.js, line 23, with NaN check. Sorry, I haven't notice that.